src/cf-buttons/src/atoms/buttons.less

```diff
@@ -72,13 +74,14 @@
   }
 
   &:hover,
-  &.hover {
+  &.hover,
+  &:focus,
```

Comment:
Although repetitive, I think it was clearer before. LESS should do the hard work of optimizing the selector use for us. Either way, I think it's kinda minor and no need to change, especially since it's so subjective.